src/scipp/__init__.py

```diff
@@ -243,6 +243,7 @@
 
 del core
 
+from .curve_fit import curve_fit
```

Comment:
Good question. Spontaneously I'd say Scipy because that's the bigger package and more people are likely familiar with it. On the other hand we are copying the Xarray api, so having the same name maybe communicates better that they are similar? I could go with either.

Reply:
@scipp/scipp-maintainers Any opinions or insights from anyone?